packages/typeahead/src/react/__specs__/utils.spec.js

```diff
@@ -0,0 +1,106 @@
+import * as utils from '../utils.js'
+
+describe('utils', () => {
```

Comment:
Starting to feel like there's a tipping point in the close future for putting these utils that have been dup'ed in several packages now into something reusable.

Reply:
100%....once we get an esmodule build that's my hope